README.md

```diff
@@ -7,27 +7,27 @@ This repository is for standards development of Interchain Standards. Initially
 #### Stage 0 - `Strawman`
 - _**Purpose**_: Start the specification process
 - _**Entrance Criteria**_: [Open an issue](https://github.com/cosmos/ics/issues/new) on this repository with a short outline of your proposal
-- _**Acceptance Signifies**_: N/A
-- _**Spec Quality**_: N/A
+- _**Acceptance Signifies**_: No acceptance required to move to the next stage. Keep the issue around to track the specification status, and close it when the final specification is merged or the proposal abandoned.
+- _**Spec Quality**_: Outline only. Link to any prior documentation, discussion, or reference materials.
 - _**Changes Expected Post-Acceptance**_: N/A
-- _**Implementation Types Expected**_: N/A
+- _**Implementation Types Expected**_: None required, but link to any existing.
 
 #### Stage 1 - `Proposal`
 - _**Purpose**_:
   * Make the case for the addition of this specification to the Cosmos ecosystem
   * Describe the shape of the a potential solution
   * Identify challenges to this proposal
 - _**Entrance Criteria**_:
-  * Prose outlining the problem or need and the general shape of a solution in a PR to a `./spec/{{ .Spec.Number }}-{{ .Spec.Name }}/proposal.md` file in this repo. This file should contain:
-    1. Illustrative examples of usage
-    1. High-level API
-    1. Discussion of key algorithms, abstractions and semantics
-    1. Identification of potential “cross-cutting” concerns and implementation challenges/complexity
+  * Prose outlining the problem or need and the general shape of a solution in a PR to a `./{{ .Spec.Number }}-{{ .Spec.Name }}/proposal.md` file in this repo. This file should contain:
+    1. Expected set of specification users along with their goals and requirements
```

Comment:
What exactly does this mean? Slightly unclear IMO

Reply:
A list of users, e.g. "Cosmos Zones" or "REST API Clients" (could include specific zones or wallets later on) - clarified the sentence.